### PR TITLE
fix: /help, /me, and /p commands can't work in non-English languages

### PR DESCRIPTION
**Pull request by ACaiCat**
### When TShock runs in a non-English language, vanilla commands prefix will be localized.
eg.(Chinese) `/help` (English command prefix) => `/帮助`  (Chinese command prefix) 

### `TShock.OnChat` cannot correctly handle vanilla command prefixes
e.g.(Chinese) When a player use `/help`, TShock will execute `/帮助` (causing the command to not be found).
image

> Invalid command entered. Type /help for a list of valid commands.

### This PR add `EnglishLanguage.GetCommandPrefixByName` to get English vanilla command prefix.
image

### CODE REVIEW

**Comment by ACaiCat** [comment]
Solve #2914

**Comment by ACaiCat** [comment]
Could someone review this PR?

**Review by hakusaro — APPROVED** [review]

**Comment by hakusaro** [comment]
Can someone else look at this too?

File: TShockAPI/Localization/EnglishLanguage.cs [hunk 1]
```diff
@@ -71,6 +75,15 @@ internal static void Initialize()
...
 				}
+
+				ChatInitializer.Load();
+				foreach (var command in ChatManager.Commands._localizedCommands)
+				{
+					if (VanillaCommandsPrefixs.ContainsKey(command.Value._name))
+						continue;
+					VanillaCommandsPrefixs.Add(command.Value._name,command.Key.Value);
+				}
+				ChatManager.Commands._localizedCommands.Clear();
```
Comment (Arthri) [inline TShockAPI/Localization/EnglishLanguage.cs:86]: This line effectively disables TShock processing Terraria commands, because TShock uses `_localizedCommands` here https://github.com/ACaiCat/TShock/blob/7419205548fb9779f83bc924222c1df9741ddff5/TShockAPI/TShock.cs#L1494

File: TShockAPI/Localization/EnglishLanguage.cs [hunk 2]
```diff
@@ -71,6 +75,15 @@ internal static void Initialize()
 					var i = (int)field.GetValue(null);
 					Prefixs.Add(i, Lang.prefix[i].Value);
 				}
+
+				ChatInitializer.Load();
+				foreach (var command in ChatManager.Commands._localizedCommands)
```
Comment (Arthri) [inline TShockAPI/Localization/EnglishLanguage.cs:80]: This doesn't seem to request the English localization?

**Review by Arthri — CHANGES_REQUESTED** [review]

**Comment by sgkoishi** [comment]
This is requesting the enUS locale as set earlier
https://github.com/Pryaxis/TShock/blob/0b6bf9ef4050f39f468d6782ea2992d8ce82e8bb/TShockAPI/Localization/EnglishLanguage.cs#L50

I guess the `Clear` was done for similar reason like my implementation (but I did set the `_localizedCommands` back after)
https://github.com/sgkoishi/yaaiomni/blob/debf82951dd84bd307fc7bce06a1c03160479976/Core/Enhancements.cs#L129

File: TShockAPI/Localization/EnglishLanguage.cs (hunk 1 above)
Comment (sgkoishi) [reply TShockAPI/Localization/EnglishLanguage.cs:86]: Just double checked and this Clear happened right before vanilla `Main.Initialize`, and it was empty before we call `ChatInitializer.Load`.

**Comment by bartico6** [comment]
> Can someone else look at this too?

I would, but I don't understand the issue very well.. Can you ELI5 what is wrong?

**Comment by sgkoishi** [comment]
> > Can someone else look at this too?
> 
> I would, but I don't understand the issue very well.. Can you ELI5 what is wrong?

Terraria will translate chat commands into command id. TShock translate them back to keep the command working.
However, when the server and the client have different locale, a `enUS` player send `/help` will be sent as `CommandId: Help` and a `deDE` server will translate it back to `/hilfe`, thus the command is broken.

**Review by Arthri — APPROVED** [review]
Tested and can confirm that it works as intended
